sites/partners/src/applications/PaperApplicationDetails/sections/DetailsPrograms.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { useContext, useMemo } from "react"
+import { t, GridSection, ViewItem, GridCell } from "@bloom-housing/ui-components"
+import { ApplicationContext } from "../../ApplicationContext"
+import { useSingleListingData } from "../../../../lib/hooks"
+
+type DetailsProgramsProps = {
+  listingId: string
+}
+
+const DetailsPrograms = ({ listingId }: DetailsProgramsProps) => {
+  const { listingDto } = useSingleListingData(listingId)
+
+  const application = useContext(ApplicationContext)
+
+  const listingPrograms = listingDto?.listingPrograms
+  const programs = application?.programs
+
+  const hasMetaData = useMemo(() => {
+    return !!listingPrograms?.filter((listingProgram) => listingProgram.program?.formMetadata)
```

Comment:
I think its better to use `.some` then `.filter` here

Reply:
Done, ty!